src/main/java/org/folio/support/ExpirationTool.java

```diff
@@ -212,14 +223,18 @@ private static Request changeRequestStatus(Request request) {
 
 
   private static Future<Set<String>> closeRequests(AsyncResult<SQLConnection> conn,
-    Vertx vertx, String tenant, List<Request> requests) {
+    Vertx vertx, String tenant, List<Request> requests, List<JsonObject> context) {
 
     Future<Void> future = succeededFuture();
     Set<String> closedRequestsItemIds = new HashSet<>();
 
     for (Request request : requests) {
+      JsonObject pair = new JsonObject();
+      pair.put(ORIGINAL.value(), JsonObject.mapFrom(request));
       closedRequestsItemIds.add(request.getItemId());
       Request updatedRequest = changeRequestStatus(request).withPosition(null);
+      pair.put(UPDATED.value(), JsonObject.mapFrom(updatedRequest));
```

Comment:
Do you need all the properties in request or just certain fields? Will it be handled properly if a new property is added?

Reply:
It is better to have all the properties for the possible future changes, new properties will not affect the result